tests/test_pipeline.py

```diff
@@ -32,6 +33,7 @@ def _test(files: List, config_path: str) -> None:
         for i in range(expected_num_transforms - 1):
             expected_transform_path = path.join(expected_file_dir, f"transformation_{i}.cpp")
             actual_transform_path = path.join(actual_file_dir, f"transformation_{i}.cpp")
+            subprocess.check_call([clang_path, actual_transform_path, "-o", path.join(actual_path, f"{i}.o")])
```

Comment:
https://docs.python.org/3/library/subprocess.html#subprocess.check_call
As I see in the documentation, if the program exit with non-zero call, then check_call raise an exception. So, we need to assert that this statement doesn't raise CalledProcessError